packages/devtools_app/test/ansi_up_test.dart

```diff
@@ -178,8 +178,10 @@ void main() {
       when(fakeServiceManager.connectedApp.isFlutterWebAppNow)
           .thenReturn(false);
       when(fakeServiceManager.connectedApp.isProfileBuildNow).thenReturn(false);
-      when(fakeServiceManager.errorBadgeManager.errorCountNotifier(any))
-          .thenReturn(ValueNotifier<int>(0));
+      // TODO(polinach): when we start supporting browser tests, uncomment
+      // and fix this. See https://github.com/flutter/devtools/issues/3616.
+      // when(fakeServiceManager.errorBadgeManager.errorCountNotifier(any))
+      //     .thenReturn(ValueNotifier<int>(0));
```

Comment:
why does browser test support affect the errorBadgeManager mocking specifically?

Reply:
Because I cannot run tis test to verify the fix. So, it is better to postpone.

Reply:
If we don't know the impact of changing these lines, it seems odd that we would comment this out then. Can we uncomment and default to leaving as it was if we can't verify?

Reply:
I commented them because analyzer complains about the parameter 'any' that is of type Null and thus cannot be passed as non-nullable parameter.

Reply:
Here's an idea that should work. Go to the MockErrorBadgeManager class and override this method:

```
@override
ValueListenable<int> errorCountNotifier(String screenId) {
    return ValueNotifier<int>(0);
  }
```

Then you can remove these lines entirely instead of commenting them out

Reply:
discussed offline and decided to comment it as is for now